js-user-data-manipulation/index.js

```diff
@@ -0,0 +1,150 @@
+// user name showing
+
+const userHeader = document.querySelector('h1');
+const input = document.querySelector('.user-form__input');
+const btn = document.querySelector('.user-form__btn');
+let username;
+
+try {
+  username = localStorage.getItem('username');
+  setUserHeader();
+} catch (error) {
+  console.log(error);
+}
+
+btn.addEventListener('click', event => {
+  event.preventDefault();
+  console.log(input.value);
+  username = input.value;
+  localStorage.setItem('username', username);
+  setUserHeader();
+});
+
+// setUserHeader();
+
+function setUserHeader() {
+  if (username) {
+    userHeader.innerText = `Hello ${username}`;
+  } else {
+    userHeader.innerText = '';
+  }
```

Comment:
You can write a little bit easier. like this `userHeader.innerText = username ? `Hello ${username}` : ''`